posts/demoting-i686-pc-windows-gnu.md

```diff
@@ -0,0 +1,43 @@
++++
+layout = "post"
+date = 2025-03-12
+title = "Demoting i686-pc-windows-gnu"
+author = "Noratrieb"
+team = "Compiler Team <https://www.rust-lang.org/governance/teams/compiler>"
++++
+
+In Rust 1.87.0, the Tier 1 target `i686-pc-windows-gnu` will be demoted to Tier 2.
+As a Tier 2 Target, builds will continue to be distributed for both the standard library and the compiler for now.
```

Comment:
```suggestion
As a Tier 2 Target, builds will continue to be distributed for both the standard library and the compiler.
```
Even if we intend to downgrade further, as a tier two target, they will be distributed going forwards.

Reply:
not if we downgrade to tier 3 :fire: (but it makes sense to remove it)